src/rtf-plugins/fixture-managers/yarpmanager/YarpFixManager.cpp

```diff
@@ -49,35 +50,31 @@ bool YarpFixManager::setup(int argc, char** argv) {
         RTF_ASSERT_ERROR_IF_FALSE(rf.check("fixture"),
                             "No application xml file is set (add --fixture yourfixture.xml)");
 
-        fixtureName = rf.find("fixture").asString();
-        std::string appfile = rf.findFileByName(std::string("fixtures/"+fixtureName).c_str());
+        manager = new YarpManagerPlugin(getDispatcher());
+        manager->fixtureName = rf.find("fixture").asString();
+
+
+        std::string appfile = rf.findFileByName(std::string("fixtures/"+manager->fixtureName).c_str());
         if (appfile.empty())
         {
             yInfo("yarpmanager: trying to load fixture file from absolute path");
```

Comment:
Not directly related to this, but if we support specifying the fixture with the absolute path I would not print something in case the user supplies an absoltue path.